src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -223,7 +223,29 @@ public function regenerate($destroy = false, $lifetime = null)
      */
     public function save()
     {
-        session_write_close();
+        try {
+            session_write_close();
+        } catch (\ErrorException $e) {
```

Comment:
this won't work in prod, as notices and warnings are *not* turned into ErrorException exceptions there.

Code must not be written by assuming that the error handler has this behavior